Documentation/building/windows-instructions.md

```diff
@@ -13,6 +13,7 @@ You must install several components to build the CoreCLR and CoreFX repos. These
 Visual Studio must be installed. Supported versions:
 - [Visual Studio 2015](https://www.visualstudio.com/vs/older-downloads/) (Community, Professional, Enterprise).  The community version is completely free.  
 - [Visual Studio 2017](https://www.visualstudio.com/downloads/) (Community, Professional, Enterprise).  The community version is completely free.  
+- [Visual Studio 2019 Preview](https://visualstudio.microsoft.com/vs/preview/) (Community, Professional, Enterprise).  The community version is completely free.  
```

Comment:
The exact workloads required should probably be figured out. It might be worth updating the VS2017 entries as well (as it specifies more than actually required in the latest release).